core/src/main/scala/org/http4s/headers/Accept-Ranges.scala

```diff
@@ -16,7 +17,33 @@ object `Accept-Ranges` extends HeaderKey.Internal[`Accept-Ranges`] with HeaderKe
   def none: `Accept-Ranges` = apply(Nil)
 
   override def parse(s: String): ParseResult[`Accept-Ranges`] =
-    HttpHeaderParser.ACCEPT_RANGES(s)
+    parser.parseAll(s).leftMap { e =>
+      ParseFailure("Invalid accept-ranges", e.toString)
+    }
+
+  /* https://tools.ietf.org/html/rfc7233#appendix-C */
+  val parser: P[`Accept-Ranges`] = {
+    val ListSep: P[Unit] = Rfc7230.ows *> P.char(',') *> Rfc7230.ows
```

Comment:
This will do a fine job on the ows-comma-ows that reasonable actors will send, but not the empty elements between commas (including leading and trailing commas) that the spec mandates.  `Rfc7230.headerRep1` should take care of those.